code/modules/reagents/chemistry/recipes/pyrotechnics.dm

```diff
@@ -411,9 +411,9 @@ datum/chemical_reaction/flash_powder
 
 /datum/chemical_reaction/shock_explosion/on_reaction(datum/reagents/holder, created_volume)
 	var/turf/T = get_turf(holder.my_atom)
-	for(var/mob/living/carbon/C in view(6, T))
+	for(var/mob/living/carbon/C in view(min(12, round(created_volume * 2)), T))
 		C.Beam(T,icon_state="lightning[rand(1,12)]",icon='icons/effects/effects.dmi',time=5) //What? Why are we beaming from the mob to the turf? Turf to mob generates really odd results.
-		C.electrocute_act(3.5, "electrical blast")
+		C.electrocute_act(min(7, created_volume), "electrical blast")
```

Comment:
leave this alone; this intention of this is to create the absolute lowest damage amount possible without it being under the thresh-hold to not stun.